### PR TITLE
Modal & grids

### PR DESCRIPTION
**Pull request by anasm11**

### CODE REVIEW

File: modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import "../general/fonts-colors.css";
+
+.modal{
+    width:350px;
```
Comment (pradipgharge) [inline modal/modal.css]: Use rems

File: styles.css [hunk 2]
```diff
@@ -1,57 +1,101 @@
...
+  color: black; 
 }
 
-body{
-    display:grid;
-    grid-auto-flow:column;
-    justify-content: center;
+/* .sidebar::-webkit-scrollbar {
+  display: none;
+} */
```
Comment (pradipgharge) [inline styles.css]: remove commented code

**Review by pradipgharge — APPROVED** [review]

File: modal/modal.css (hunk 1 above)
Comment (anasm11) [reply modal/modal.css]: okay..i'll change that

**Review by varun1233 — COMMENTED** [review]
Code looks good to merge.
UI Feedback:
Your aside bar is breaking
image

**Comment by anasm11** [comment]
> Code looks good to merge. UI Feedback: Your aside bar is breaking image

Thank you. I'll fix that.

**Pull request merged**